File: sources/easygenerator.Infrastructure/DomainModel/Mappings/OrganizationInvite.cs
using System;

namespace easygenerator.Infrastructure.DomainModel.Mappings
{
    public class OrganizationInvite
    {
        public Guid Id { get; set; }
        public Guid OrganizationId { get; set; }
        public string OrganizationTitle { get; set; }
        public string OrganizationAdminFirstName { get; set; }
        public string OrganizationAdminLastName { get; set; }
    }
}

File: sources/easygenerator.Web/App/models/organizations/organizationInvite.js
define(['models/entity'],
   function (EntityModel) {
       "use strict";

       var Organization = function (spec) {
           var obj = new EntityModel(spec);

           obj.organizationId = spec.organizationId;
           obj.organizationAdminFirstName = spec.organizationAdminFirstName;
           obj.organizationAdminLastName = spec.organizationAdminLastName;
           obj.organizationTitle = spec.organizationTitle;
           obj.status = spec.status;

           return obj;
       };

       return Organization;
   }
);